docs/source/query-syntax.rst

```diff
@@ -0,0 +1,135 @@
+.. _query-syntax:
+
+Search
+======
+
+MLflow UI and API supports searching runs within a single experiment or a group of experiments
+using a search filter API. This API is simplified version of WHERE clause of SQL syntax.
+
+.. contents:: Table of Contents
+  :local:
+  :depth: 2
+
+Syntax Description
+------------------
+
+Search filter can be a single expression or several expressions separated by ``AND`` keyword.
+Currently, the syntax does not support ``OR``. Each expression has 3 parts: an identifier on
+the LHS, a comparator, and constant on the RHS.
+
+Identifier
+^^^^^^^^^^
+
+Left hand side of a search expressions, signifies a logged entity to compare against. It has two
+sub-parts, a qualifier for and entity you are searching and the name of the entity, separated by
+a period. The qualifier or type of entity you can search can be one of `metrics`, `params`, or
+`tags`.
+
+Example: ``metrics.accuracy``
+
+Constant
+^^^^^^^^
+
+Search syntax requires the RHS of the expression to be a constant. Type of expected constant on
+depends on LHS.
+ - If LHS of the expression is a metric, RHS is expected to be a number (integer or float)
+ - If LHS is a parameter or tag, RHS is expected to be a string constant
```

Comment:
Nit: Can we explicitly stated that the string constant must be quoted in either single or double quotes?

Reply:
I'm editing the file and can make this fix.

Reply:
@mparkhe  None of your examples have the constants in quotes. If they must be in quotes, how can the examples work?

Reply:
Ignore my previous comment.

Reply:
Thanks @stbof